pages.it/common/bastet.md

```diff
@@ -0,0 +1,33 @@
+# bastet
+
+> Clone di Tetris nel Terminale.
+> I tasti sono personalizzabili, i valori predefiniti sono di seguito.
```

Comment:
This is not in the main page. The keys are customizable in a lot of apps, I think this is not needed, who customize the keys don't need this page.
```suggestion
```